src/bin/utils.ts

```diff
@@ -196,6 +207,35 @@ async function requestPassword(): Promise<string> {
 //   return;
 // }
 
+const envVariables = {
+  PK_PASSWORD: process.env.PK_PASSWORD,
+  PK_RECOVERY_CODE: process.env.PK_RECOVERY_CODE,
+}
```

Comment:
Does having this inside `src/bin/utils` make importing this file side-effectful? Isn't this what we were trying to avoid by removing this from `src/config,ts`? Need some clarification on this @CMCDragonkai 

My expectation was that `process.env` would only be found in the command within `src/bin` (e.g. `src/bin/agent/start`, etc). In that case, it's a good idea to keep the environment variable constants somewhere though, and `src/bin/utils` sounds as good a place as any for that though.

Reply:
I don't know the details of what problems this might have. It seems to work when I checked it but i'll double check.
From what I recall all i saw about having it in src/config was that we wanted everything in there to be static.

Reply:
So initially we were planning to have the `src/config.ts` contain all configuration details (including all environment variables). See https://github.com/MatrixAI/js-polykey/issues/194#issuecomment-952524132. But this induces a side-effect, such that whenever you import the file, it performs this side-effect of reading from the environment variables https://github.com/MatrixAI/js-polykey/issues/194#issuecomment-952547227

As I understand it, any retrieval of environment variables should be strictly in `src/bin/<command>`

Reply:
As per the spec, I imagine a `getFromEnv` function can be used here. e.g. a basic prototype

```ts
async function getFromEnv(envVar: string): string | undefined {
  return process.env[envVar]; // or process.env.envVar - whatever it may be
}
```

This way, you're not implicitly importing the environment variables every time you import the file. I believe this would solve this issue? @CMCDragonkai is more aware of the side-effects of environment variables here.

Reply:
Yea it's better to do this from within the scripts. That means while the expression can be written in the `utils.ts`, they are only meant to be executed in the bin scripts.

And `src/config.ts` is meant to be static now.

Reply:
Although I realise that by using `option.env`, it may actually read the env variable at that point. Although I reckon that would be a mistake upstream and they should be reading only when the option is being parsed.

Reply:
I'll make them functions but I don't think it makes a difference.